data/sql/updates/pending_db_world/rev_1614870494129873100.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1614870494129873100');
+
+DELETE FROM `spell_target_position` WHERE `ID`=62501 AND `EffectIndex`=0;
+INSERT INTO `spell_target_position` (`ID`, `EffectIndex`, `MapID`, `PositionX`, `PositionY`, `PositionZ`, `Orientation`, `VerifiedBuild`) VALUES (62501, 0, 603, 2035.95, -202.085, 432.687, 3.16408, 0);
+
+DELETE FROM `spell_script_names` WHERE spell_id = 62502;
```

Comment:
Backticks

Reply:
backticks cause an error with mysql for some reason

Reply:
@Kitzunu 

> ERROR 1054 (42S22) at line 10963: Unknown column 'spell_id ' in 'where clause'

[this commit called continuation](https://github.com/azerothcore/azerothcore-wotlk/pull/4703/commits/f389bcd536ccb40504bc259fe1ca64e6ebc2694a)

Reply:
not for me

Reply:
![image](https://user-images.githubusercontent.com/24550914/110215564-bddf4c00-7eaa-11eb-828c-37c39468b99d.png)
found your error, a space :)

Reply:
!!!!!!!!!!!!!!!!!!!!!!